eth/vm/forks/muir_glacier/__init__.py

```diff
@@ -0,0 +1,31 @@
+from typing import (
+    Type,
+)
+
+from eth.rlp.blocks import BaseBlock
+from eth.vm.forks.istanbul import (
+    IstanbulVM,
+)
+from eth.vm.state import BaseState
+
+from .blocks import MuirGlacierBlock
+from .headers import (
+    compute_muir_glacier_difficulty,
+    configure_muir_glacier_header,
+    create_muir_glacier_header_from_parent,
+)
+from .state import MuirGlacierState
+
+
+class MuirGlacierVM(IstanbulVM):
+    # fork name
+    fork = 'muir-glacier'
+
+    # classes
+    block_class: Type[BaseBlock] = MuirGlacierBlock
```

Comment:
We could start using `ClassVar` for these if you want to be fancy.  I'm not sure it gains us much since I don't think we would be likely to write any code that would overwrite them but maybe still worth having it enforced by the type checker....

Reply:
That's probably a good idea but we would have to apply that all the way up to the very top of the inheritance chain because otherwise you run into:

>Cannot override instance variable (previously declared on base class "IstanbulVM") with class variable

Maybe we keep that out of this PR and apply it as a separate refactoring.